Assets/Python/CityNameManager.py

```diff
@@ -3857,12 +3893,14 @@ def onRevolution(iCiv):
 	"Kabura"		:	"Kabul",
 	"Kaesong"		:	"Kaesong",
 	"Kaijo"			:	"Kaesong",
-	"Songak"		:	"Kaesong",
+	"Kaesong"		:	"Kaesong",

```

Comment:
What happened to Songak? `Kaesong -> Kaesong` twice seems like a mistake.

Reply:
Removed one of the duplicated lines.
The proper name is Kaesong,
Songak was more like a nickname and was used officially for only a short time.